testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -69,12 +68,10 @@ public static void exposeFixedPorts(GenericContainer<?> container)
                 "This method is supposed to be invoked from local development helpers only e.g. QueryRunner.main(), " +
                 "hence it should never run on CI");
 
-        ImmutableList<PortBinding> portBindings = container.getExposedPorts().stream()
-                .map(exposedPort -> new PortBinding(bindPort(exposedPort), new ExposedPort(exposedPort)))
-                .collect(toImmutableList());
-
         container.withCreateContainerCmdModifier(cmd -> cmd
                 .withHostConfig(requireNonNull(cmd.getHostConfig(), "hostConfig is null")
-                        .withPortBindings(portBindings)));
+                        .withPortBindings(container.getExposedPorts().stream()
+                                .map(exposedPort -> new PortBinding(bindPort(exposedPort), new ExposedPort(exposedPort)))
+                                .collect(toImmutableList()))));
```

Comment:
```
Fix for method exposing port on local environment 

This fix prevents loosing ports which were exposed
after we call this method
```

I couldn't understand what was wrong. I ran PostgreSqlQueryRunner with/without this commit, but the result looks same to me. Could you describe more details?

Reply:
@ebyhr 
Call to this method add some override rules, which will be executed later.
So it's possible such situation.
1) we collect current expose ports by calling this method (but not yet execute it)
2) .... add more expose ports in other code ....
3) actually apply overrides but only with values from 1) missing values from 2)

With Postgres it works because all exposed ports was added before 1)